main.nf

```diff
@@ -554,7 +557,7 @@ if(params.pseudo_aligner == 'salmon' && !params.salmon_index){
 
         script:
         """
-        salmon index --threads $task.cpus -t $fasta -i salmon_index
+        salmon index --threads $task.cpus -t $fasta --gencode -i salmon_index
```

Comment:
What happens if it is not in gencode format? Does it fail then? 

Or could we have something like `params.salmon_gencode` as a parameter to turn on/off this behavior?

Reply:
it seems if the name is separated with spaces (as standard) it should work the same. I am ok with an extra parameter to be safe, as well.